tools/alignmentfile_pairedness.cwl

```diff
@@ -27,6 +27,10 @@ requirements:
         }
   - class: ResourceRequirement
     coresMin: $(inputs.cpu)
+    ramMin: $(inputs.ram * 1000)
+    https://platform.illumina.com/rdf/ica/resources:tier: economy
```

Comment:
Ooof, we gotta shove this in each tool for ICA?

Reply:
I'm currently trying to get answers on this but, based on my testing, yes. I originally tried just setting this once at the workflow level hoping it would be inherited by all of the tools, but that didn't work. Behavior-wise it seems like if you set `coresMin` and `ramMin` it would overwrite the workflow-level tier hint.

Hopefully I can get more information out of the Illumina team in the coming days.